src/Backend/Core/Layout/Templates/FormLayout.html.twig

```diff
@@ -370,9 +370,15 @@
   {% if form.parent.meta is defined %}
     {% set meta = form.parent.meta %}
     {% set slug = meta.children.url.vars.value %}
-    <a target="_blank" rel="noopener noreferrer" class="site-url" href="{{ meta.vars.detail_url }}/{{ slug }}">
-      {{ SITE_URL }}{{ meta.vars.detail_url }}/<span id="generatedUrl">{{ slug }}</span>
-    </a>
+    {% if slug == 'home' %}
+      <a target="_blank" rel="noopener noreferrer" class="site-url" href="{{ meta.vars.detail_url }}/">
```

Comment:
`/` is not needed in href?

Reply:
@Katrienvh fixed c098435854e65e3209130c0a70c661fd6f58250b